opm/parser/eclipse/EclipseState/Grid/GridProperty.hpp

```diff
@@ -103,7 +103,9 @@ class GridProperty {
     void iset(size_t index, T value);
     void iset(size_t i , size_t j , size_t k , T value);
 
+
     const std::vector<T>& getData() const;
+    std::vector<T>& getData();
```

Comment:
> Looks like OPERATE can modify grid properties as well, so I'd wager it's to facilitate that.

Yes that is right - the other keywords which can modify a property, like e.g. `COPY` and `MULT` are implemented with mutating property methods. In this case the required property method had to have a quite weird signature and the implementation is therefor applied directly to the underlying `std::vector<T>`.